data/stories.md

```diff
@@ -46,4 +46,46 @@
 * out_of_scope
     - utter_out_of_scope
     - open_incident_form
-    - form{"name":null}
\ No newline at end of file
+    - form{"name":null}
+
+## story_version
+* version
+  - utter_version
+  - action_version
+
+## f1_score
+* f1_score
+  - action_f1_score
+
+## show_slots
+* show_slots
+  - action_show_slots
```

Comment:
I'm not sure these kind of debugging stories should be included since it is a demo repo. It's not something recommended for people to include in their bots is it?

Reply:
I can remove the `show_slots` and `f1_score` features but I think the audience for this demo includes developers who would find value in admin/debug features.

Reply:
Ja maybe that's a happy medium then

Reply:
(keeping the version, that is)